dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf

```diff
@@ -1,3 +1,17 @@
+resource "kafka_topic" "genesys_eb_events" {
+  name = "contact-channels.genesys_eb_events"
+
+  replication_factor = 3
+  partitions         = 9
+
+  config = {
+    "retention.ms"      = "2629800000" # 1 month
```

Comment:
Would it be ok if we enable tiered storage for this topic, and only keep, let's say 3 days into local storage?
Details and an example is at point 5 in https://github.com/utilitywarehouse/kafka-cluster-config/tree/main/dev-aws/kafka-shared-msk#contributing

Reply:
@sbuliarca If I understand correctly if we enable that it means that the data will be available in `hot` storage (aka faster reading speeds) for 3 days and then moved to low-cost storage (slower reading speeds at least at the beginning of reading) up to the max of 1month as I have set here?

Reply:
Exactly

Reply:
@sbuliarca I have added the `local.retention.ms` option. Is that the only one? I saw some people had `remote.storage.enable` so I was not sure if that is needed as well.

Reply:
You need the `remote.storage.enable` too

Reply:
@sbuliarca Thank you. Added it now.